xml/src/main/java/org/ehcache/xml/service/DefaultCacheEventListenerConfigurationParser.java

```diff
@@ -78,6 +79,9 @@ public CacheType unparseServiceConfiguration(CacheConfiguration<?, ?> cacheConfi
 
       Set<ListenersType.Listener> listeners = serviceConfigs.stream().map(serviceConfig -> {
         ListenersType.Listener listener = new ListenersType.Listener();
+        if (serviceConfig.getInstance() != null) {
```

Comment:
This logic is repeated in multiple classes that extends `SimpleCoreServiceConfigurationParser`. It can be moved to `SimpleCoreServiceConfigurationParser.unparseServiceConfiguration`.

Reply:
It's repeated in those classes that don't extend SimpleCoreServiceConfigurationParser and have separate unparsing functions.

Reply:
Just the copier and serializer configs don't extend from it. Let the duplication be there. But it can still be avoided from the other two.